Build fails locally

## Expected Behavior
```bash 
# should work without error
make build
docker build .
```
## Actual Behavior
```
Sending build context to Docker daemon  12.67MB
Step 1/9 : FROM golang:1.12.5
 ---> 1ef078f0da9e
Step 2/9 : COPY . /go/src/github.com/grafeas/grafeas/
 ---> a93781e4f733
Step 3/9 : WORKDIR /go/src/github.com/grafeas/grafeas/samples/server/go-server/api/server/main
 ---> Running in 6fa63b365058
Removing intermediate container 6fa63b365058
 ---> 595481c975ec
Step 4/9 : RUN CGO_ENABLED=0 go build -o grafeas-server .
 ---> Running in 5ccc054b7a96
client.go:7:2: cannot find package "github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto" in any of:
	/go/src/github.com/grafeas/grafeas/vendor/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (vendor tree)
	/usr/local/go/src/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (from $GOROOT)
	/go/src/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (from $GOPATH)
../api/api.go:30:2: cannot find package "github.com/grafeas/grafeas/proto/v1beta1/project_go_proto" in any of:
	/go/src/github.com/grafeas/grafeas/vendor/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (vendor tree)
	/usr/local/go/src/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (from $GOROOT)
	/go/src/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (from $GOPATH)
The command '/bin/sh -c CGO_ENABLED=0 go build -o grafeas-server .' returned a non-zero code: 1
Makefile:42: recipe for target 'build-docker-image' failed
make: *** [build-docker-image] Error 1
```

## Steps to Reproduce the Problem
clone repo locally, run make or docker build...

Same issue for me:

During a docker build I'm obtaining:


```
Sending build context to Docker daemon  9.089MB
Step 1/9 : FROM golang:1.12.5
 ---> 1ef078f0da9e
Step 2/9 : COPY . /go/src/github.com/grafeas/grafeas/
 ---> Using cache
 ---> aae0d13ff938
Step 3/9 : WORKDIR /go/src/github.com/grafeas/grafeas/samples/server/go-server/api/server/main
 ---> Using cache
 ---> 340fb3382f1d
Step 4/9 : RUN CGO_ENABLED=0 go build -o grafeas-server .
 ---> Running in 6534dba1499f
client.go:7:2: cannot find package "github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto" in any of:
        /go/src/github.com/grafeas/grafeas/vendor/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (vendor tree)
        /usr/local/go/src/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (from $GOROOT)
        /go/src/github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto (from $GOPATH)
../api/api.go:30:2: cannot find package "github.com/grafeas/grafeas/proto/v1beta1/project_go_proto" in any of:
        /go/src/github.com/grafeas/grafeas/vendor/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (vendor tree)
        /usr/local/go/src/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (from $GOROOT)
        /go/src/github.com/grafeas/grafeas/proto/v1beta1/project_go_proto (from $GOPATH)
The command '/bin/sh -c CGO_ENABLED=0 go build -o grafeas-server .' returned a non-zero code: 1
```

My environment looks like the following:
OS: OS X Mojave (10.14.2)
Golang Version: 1.12.5 darwin/amd64
Docker Version: 18.09.2, build 6247962

My GOPATH contains `/Users/<username>/go` and my grafeas is located under `/Users/<username>/go/src/github.com/grafeas/grafeas` (where the root of the project lies)

Running `make build` produces this output:
```bash
go clean ./...
rm -rf *~ .install.tools protoc proto/*/*_go_proto
mkdir -p protoc
curl https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-osx-x86_64.zip -o protoc/protoc.zip -L
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   162    0   162    0     0    946      0 --:--:-- --:--:-- --:--:--   941
100   617    0   617    0     0   2289      0 --:--:-- --:--:-- --:--:--  2289
100 1417k  100 1417k    0     0  1291k      0  0:00:01  0:00:01 --:--:-- 1291k
unzip protoc/protoc -d protoc
Archive:  protoc/protoc.zip
   creating: protoc/include/
   creating: protoc/include/google/
   creating: protoc/include/google/protobuf/
  inflating: protoc/include/google/protobuf/any.proto
  inflating: protoc/include/google/protobuf/api.proto
   creating: protoc/include/google/protobuf/compiler/
  inflating: protoc/include/google/protobuf/compiler/plugin.proto
  inflating: protoc/include/google/protobuf/descriptor.proto
  inflating: protoc/include/google/protobuf/duration.proto
  inflating: protoc/include/google/protobuf/empty.proto
  inflating: protoc/include/google/protobuf/field_mask.proto
  inflating: protoc/include/google/protobuf/source_context.proto
  inflating: protoc/include/google/protobuf/struct.proto
  inflating: protoc/include/google/protobuf/timestamp.proto
  inflating: protoc/include/google/protobuf/type.proto
  inflating: protoc/include/google/protobuf/wrappers.proto
   creating: protoc/bin/
  inflating: protoc/bin/protoc
  inflating: protoc/readme.txt
cd tools && GO111MODULE=on go install -v github.com/golang/protobuf/protoc-gen-go
cd tools && GO111MODULE=on go install -v github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
cd tools && GO111MODULE=on go install -v github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
protoc/bin/protoc -I ./ -I vendor/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis -I vendor/github.com/grpc-ecosystem/grpc-gateway -I vendor/github.com/googleapis/googleapis \
		--go_out=plugins=grpc,paths=source_relative:. \
		--grpc-gateway_out=logtostderr=true,paths=source_relative:. \
		proto/v1beta1/attestation.proto
protoc-gen-go: program not found or is not executable
--go_out: protoc-gen-go: Plugin failed with status code 1.
make: *** [proto/v1beta1/attestation_go_proto/.done] Error 1
```

In the `protoc/bin/protoc` step of the Makefile I actually changed the curl command to retrieve the osx version of protoc which got me here.

Hope that provides some more insight

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,8 @@
 FROM golang:1.12.5
+RUN apt-get update && apt-get install unzip
 COPY . /go/src/github.com/grafeas/grafeas/
+WORKDIR /go/src/github.com/grafeas/grafeas
+RUN make build
 WORKDIR /go/src/github.com/grafeas/grafeas/go/v1beta1/main
 RUN CGO_ENABLED=0 go build -o grafeas-server .
 

